projects/ccd-case-ui-toolkit/src/lib/shared/pipes/complex/ccd-read-fields-filter.pipe.ts

```diff
@@ -62,8 +62,9 @@ export class ReadFieldsFilterPipe implements PipeTransform {
   }
 
   private static isEmpty(value: any): boolean {
-    return ReadFieldsFilterPipe.EMPTY_VALUES.indexOf(value) !== -1
-      || value.length === 0;
+    const valueIsEmpty = value?.hasOwnProperty('list_items')  && value?.hasOwnProperty('value') ?
+      ReadFieldsFilterPipe.EMPTY_VALUES.indexOf(value.value) !== -1 : ReadFieldsFilterPipe.EMPTY_VALUES.indexOf(value) !== -1;
```

Comment:
Kinda nasty - had to read it a few times to spot why you have ReadFieldsFilterPipe.EMPTY_VALUES.indexOf() twice
maybe just do this 
const fieldValue = value?.list_items && value?.value ? value.value : value;
const valueIsEmpty =  ReadFieldsFilterPipe.EMPTY_VALUES.indexOf(fieldValue);
return valueIsEmpty || fieldValue.length == 0;